src/js/components/CheckBoxGroup/README.md

```diff
@@ -87,4 +87,15 @@ string
 
 ```
 div
-```
\ No newline at end of file
+```
+## Theme
+  
+**checkBoxGroup.container**
+
+Any valid Box props for the RadioButtonGroup container. Expects `object`.
```

Comment:
Probably best to re-run the generate-docs command to capture this and the corresponding snapshot.